lib/thor.rb

```diff
@@ -66,15 +66,23 @@ def desc(usage, description, options = {})
 
     # Defines the long description of the next command.
     #
+    # Long description is by default indented, line-wrapped and repeated whitespace merged.
+    # In order to print long description verbatim, with indentation and spacing exactly
+    # as found in the code, use the +wrap+ option
+    #
+    #   long_desc 'your very long description', wrap: false
+    #
     # ==== Parameters
     # long description<String>
+    # options<Hash>
     #
     def long_desc(long_description, options = {})
       if options[:for]
         command = find_and_refresh_command(options[:for])
         command.long_description = long_description if long_description
       else
         @long_desc = long_description
+        @long_desc_wrap = options[:wrap] || options[:wrap].nil?
```

Comment:
```suggestion
        @long_desc_wrap = options[:wrap] != false
```

Reply:
I agree that the proposed change makes the code look cleaner and easier to read.